getting-started/coaddition.rst

```diff
@@ -42,180 +46,89 @@ There are two general categories of sky maps:
 
 1. Whole sky.
 2. A selected region containing a set of exposures.
```

Comment:
No this is describing a `DiscreteSkyMap`.  That takes as input a list of exposures and builds a bounding box around them.  That ends up being the tract boundary and the patches are boxes within that bounding box.  I'll see if I can clarify

Reply:
So if you made one with one exposure it would be a very small skymap? With multiple calexps I'm still not entirely sure how you work out what the discrete sky map looks like if some overlap with each other and some don't but I'm aware that this tutorial can't explain in detail what a discrete sky map is.